### DISCUSSION TITLE
Recovery

### DISCUSSION THREAD

**Discussion by bajipogadadanda** [webdriverio/codemod#107] [General]
Hi,
I would like to know if Helenium or similar mechanism can be used for WDIO? It helps to improve the reliability.

**Comment by christian-bromann** [comment]
WebdriverIO has already auto-waits for commands and assertions implemented. There should be no need for the use of a tool like this. Are there any issues you are facing particularly or are you looking for a specific feature?

**Reply by bajipogadadanda** [reply]
Usually, we get  NoSuchElement test failures if there are any small changes for the locators, I am trying to implement In runtime that replacing failed control with a new value that matches the best and performs an action with the replaced control successfully.
healenium helps for it, very long back its been attempted for Selenium/TestNG, So, I thought to confirm if it supports WebdriverIO also, I read from  helenium site that it supports WebdriverIO also.

**Reply by christian-bromann** [reply]
Have you tried following the docs: https://healenium.io/frameworks#rec639832796

**Comment by bajipogadadanda** [comment]
Thx, let me try
